### DISCUSSION TITLE
Zipformer encoder, written description

### DISCUSSION THREAD

**Discussion by KarelVesely84** [k2-fsa/sherpa#397] [General]
Hello,
I am going through the zipformer source code, to try to understand it better.
Is there any chance that it was described somewhere recently ?

I tried the publications of Dan Povey here: 
- https://dblp.org/pid/35/4478.html

And also there was some info in the videos: 
- https://www.youtube.com/watch?v=mVU259EmxH4
- https://www.youtube.com/watch?v=Q3gNj7XlArs&t=1s

Is there any chance that there would be some other written source somewhere ?
Best regards
Karel

**Comment by danpovey** [comment]
Sorry, we are working on a description, but there is no publication so far
